modules/facilities_api/spec/spec_helper.rb

```diff
@@ -2,14 +2,16 @@
 
 # Configure Rails Envinronment
 ENV['RAILS_ENV'] = 'test'
-# require File.expand_path('dummy/config/environment.rb', __dir__)
+require File.expand_path('dummy/config/environment.rb', __dir__)
```

Comment:
Does this file exist anywhere? Trying to figure out how where `dummy/config/environment.rb` lives.

Reply:
Not really. I was trying to get specs to run from within the engine and forgot to recommend it.
you can see that the [generator](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/generators/module/templates/spec/spec_helper.rb.erb#L5) we are using adds that in as well